lib/Customer.php

```diff
@@ -265,4 +266,55 @@ public static function allTaxIds($id, $params = null, $opts = null)
     {
         return self::_allNestedResources($id, static::PATH_TAX_IDS, $params, $opts);
     }
+
+    /**
+     * @param string|null $id The ID of the customer on which to create the customer balance transaction.
```

Comment:
```suggestion
     * @param string|null $id The ID of the customer on which to create the balance transaction.
```

And same for the comments for the other methods.